apps/docs/src/app/core/component-docs/list/list-docs.component.html

```diff
@@ -1,16 +1,78 @@
-<fd-docs-section-title [id]="'list-simple'" [componentName]="'list'">
-    Simple List
+<fd-docs-section-title id="list-simple" componentName="list">
+    Simple list
 </fd-docs-section-title>
-<description>A simple list. The list item can contain plain text or links.</description>
+<description>
+    To create a simple list extend your HTML list implementation with following directives: <code>fd-list</code>, <code>fd-list-item</code>,
```

Comment:
"To display a simple list, extend your HTML list implementation with the following directives:

-  <code>fd-list</code>
- <code>fd-list-item</code>
- <code>fd-list-title</code>

"

This is easier for the users to scan